### PR TITLE
PUBDEV-8998: Fix numpy in fairness

### PR DESCRIPTION
**Pull request by tomasfryda**
https://h2oai.atlassian.net/browse/PUBDEV-8998

### CODE REVIEW

File: h2o-py/h2o/model/extensions/fairness.py [hunk 1]
```diff
@@ -191,6 +194,9 @@ def fair_roc_plot(self, frame, protected_columns, reference, favorable_class, fi
         >>> gbm.fair_roc_plot(test, protected_columns, reference, favorable_class)
         """
         import h2o
+        from h2o.utils.shared_utils import can_use_numpy
+        if not can_use_numpy():
+            raise ImportError("numpy is required for fair_roc_plot.")
```
Comment (tomasfryda) [inline h2o-py/h2o/model/extensions/fairness.py:199]: Doesn't use `numpy` directly but indirectly so it's not imported in this function but it is required.

File: h2o-py/h2o/model/extensions/fairness.py [hunk 2]
```diff
@@ -254,7 +260,9 @@ def fair_pr_plot(self, frame, protected_columns, reference, favorable_class, fig
         """
         import h2o
         from h2o.utils.typechecks import assert_is_type
-
+        from h2o.utils.shared_utils import can_use_numpy
+        if not can_use_numpy():
+            raise ImportError("numpy is required for fair_pr_plot.")
```
Comment (tomasfryda) [inline h2o-py/h2o/model/extensions/fairness.py:265]: Doesn't use `numpy` directly but indirectly so it's not imported in this function but it is required.

File: h2o-py/h2o/model/extensions/fairness.py [hunk 3]
```diff
@@ -420,6 +432,9 @@ def inspect_model_fairness(self, frame, protected_columns, reference, favorable_
         >>> gbm.inspect_model_fairness(test, protected_columns, reference, favorable_class)
         """
         import h2o
+        from h2o.utils.shared_utils import can_use_numpy
+        if not can_use_numpy():
+            raise ImportError("numpy is required for inspect_model_fairness.")
```
Comment (tomasfryda) [inline h2o-py/h2o/model/extensions/fairness.py:437]: Doesn't use `numpy` directly but indirectly so it's not imported in this function but it is required.

**Review by wendycwong — APPROVED** [review]
Thank you for the quick fix.

**Review by mn-mikke — APPROVED** [review]
LGTM, thank you!

**Comment by ledell** [comment]
Thanks @tomasfryda!

**Review by ledell — APPROVED** [review]
Looks good.

**Pull request merged**